paasta_tools/kubernetes_tools.py

```diff
@@ -586,7 +586,7 @@ def get_autoscaling_metric_spec(
 
         min_replicas = self.get_min_instances()
         max_replicas = self.get_max_instances()
-        if min_replicas is None or max_replicas is None:
+        if not min_replicas or not max_replicas:
```

Comment:
Can we either add a comment saying that we are handling both 0 and None here or (even better) make that explicit somehow in the code by checking for both None and 0?

My impulse is usually the same as yours -- when I see something relying on implicit conversion to boolean I switch it to `is not None`, but here we are (implicitly) dealing with 0s.

Reply:
OK, I separated the checks.

get_min_instances returns `get('min_instances', 1)`, so it shouldn't be None. And `is_autoscaling_enabled` checks if max_instances is 0.